tests/tamr_client/fake_json/dataset/test_record/test_stream.json

```
@@ -0,0 +1,13 @@
+[
+    {
+        "request": {
+            "method": "GET",
+            "path": "datasets/1/records"
+        },
+        "response": {
+            "ndjson": [
+                {"primary_key": 1}, {"primary_key": 2}
```

nit: can we newline-separate these items to be closer to the `ndjson` format?

```suggestion
                {"primary_key": 1}, 
                {"primary_key": 2}
```